src/labels/CustomLabels.labels

```diff
@@ -6065,6 +6065,24 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Failure message if an invalid base64 url was provided</shortDescription>
         <value>The base64url format is invalid. It can only contain these characters: [a-z A-Z 0-9 _ -]</value>
     </labels>
+    <labels>
+        <fullName>gePaymentProcessError</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Failure message description for tokenization or purchase errors</shortDescription>
+        <value>There was an error processing the transaction. Please review the error and try again. Alternatively,
+            select Do Not Charge Card to enter a pledged donation without processing a card payment. {0} errors:
+            /newline/ {1} </value>
```

Comment:
@melissabarber just a quick comment on this one. the '/newline/' character there won't appear in the custom label. this is so we can know where we want to add a line break since we have a single custom label that will display on two lines.

Reply:
I think the /newline/ character might have been an artifact. The screenshot you sent in Slack looks like the correct formatting with the {1} variable of the Elevate error messages appearing below this error as sub-bullets.

Reply:
@melissabarber that's correct. It won't show in the UI. I'm using it just to know where we need to break the line. So it's only being used by the code.